packages/typescriptlang-org/src/copy/zh/community.ts

```diff
@@ -0,0 +1,27 @@
+export const comCopy = {
+  com_layout_title: "如何配置TypeScript",
+  com_layout_description: "与其他Typescripters在线或离线交流",
+  com_headline: "联系我们",
+  com_connect_online: "在线",
+  com_connect_online_description:
+    " 把那些做得好的，你想添加或者改进的地方告诉我们，以及获得新的更新。",
+  com_online_stack_overflow_desc:
+    "与您的同龄人互动，并使用'typescript'标签询问有关TypeScript的问题",
+  com_online_stack_overflow_tag: "",
+  com_online_discord_header: "Chat",
+  com_online_discord_desc: "在TypeScript社区聊天中与其他TypeScript用户聊天。",
+  com_online_github_desc: "找到了bug, 或者想要给我提建设性建议？",
+  com_online_github_href: "在Github上告诉我们",
+  com_online_twitter_desc: "保持最新的消息。在Twitter上关注我们",
+  com_online_blog_desc: "了解最新的Typescript的进展，通过我们的",
```

Comment:
Not sure why these two does not have corresponding `_href` like `com_online_github_href`. And also should have better templating so Chinese translation can be more fluent, since here the href part should be placed in middle of the sentence.

Reply:
you are right, I think it would be better if we have some kind of tempates that can accept  parameters, like: 

`Learn about the latest TypeScript developments via our %{link}`

Reply:
I'm definitely open to updates on the English version which make it easier to translate! These are hard to figure out ahead of time. It's all based on https://formatjs.io/docs/getting-started/installation/